packages/buckets/src/utils.ts

```diff
@@ -24,10 +23,10 @@ export async function listPathRecursive(
   path: string,
   depth: number,
   currentDepth = 0,
-) {
+): Promise<ListPathReply.AsObject> {
```

Comment:
better build

Reply:
Yeh, we should just _always_ do this for external APIs. I have a linter setting coming in that'll make sure we do.